README.md

```diff
@@ -23,6 +26,37 @@ This repository contains an SDK and Jupyter notebooks with examples to showcase
 - [Build Status](#BuildStatus)
 - [Additional References](#Refs)
 
+# <a name="intro"></a> 
+
+# Overview of Interpret-Community
+Interpret-community is a practical and community-driven repository that hosts a wide range of community-developed machine learning interpretability techniques. This repository makes it easy for anyone involved in the development of an AI system to achieve transparency around their machine learning models. Data scientists, machine learning engineers, and researchers can easily add their own interpretability techniques via the set of extension hooks built into the parent repository, Interpret, and expand this repository to include their custom-made interpretability techniques.

```

Comment:
I'd suggest removing the - in community-driven and community-developed.
Separate note, the community is not driving so.....
Suggest change AI to ML, change achieve transparency to improve transparency